contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -74,6 +76,7 @@ const (
 	defaultMesosUser         = "root" // should have privs to execute docker and iptables commands
 	defaultReconcileInterval = 300    // 5m default task reconciliation interval
 	defaultReconcileCooldown = 15 * time.Second
+	defaultNodeRelistPeriod  = 5 * time.Minute
```

Comment:
might be nice to make this tunable, either via flag or scheduler config file (for large clusters, this value is probably too small)